src/input/mod.rs

```diff
@@ -675,6 +676,8 @@ impl State {
 
                     position += event.delta().as_global();
 
+                    // RYANABX LOOK HERE
+
```

Comment:
Well, if I am supposed to merge this, you might need to cleanup some left-over comments. ;)

Reply:
oops :laughing:

Reply:
fixed in 0fd688dae601937a723586e37bc1d90359a47e9e